Userland/Applications/KeyboardSettings/KeyboardSettingsWidget.cpp

```diff
@@ -172,22 +189,28 @@ KeyboardSettingsWidget::KeyboardSettingsWidget()
     m_remove_keymap_button->on_click = [&](auto) {
         auto& selection = m_selected_keymaps_listview->selection();
         for (auto& index : selection.indices()) {
+            bool active_keymap_deleted;
+            if (keymaps_list_model.keymap_at(index.row()) == m_active_keymap_label->text())
+                active_keymap_deleted = true;
             keymaps_list_model.remove_at(index.row());
+            if (active_keymap_deleted)
+                m_active_keymap_label->set_text(keymaps_list_model.keymap_at(0));
```

Comment:
Right. This List View uses the default `SelectionMode`, which is `SingleSelection`, so there's only ever a single selection and I was counting on that. But your suggestion would make the code clearer and future-proof if multiple selections are allowed, so I updated as you suggested :).

Also, there should always be a `keymap_at(0)` since the remove button gets deactivated when there's a single keymap left on the list.

Reply:
Got it, yeah seems fine then (the `keymap_at(0)` thing).